addb2/dump.c

```diff
@@ -422,7 +422,7 @@ static void ptr(struct m0_addb2__context *ctx, const uint64_t *v, char *buf)
 {
 	if (json_output)
 		/* JSON spec supports only decimal format (int and float) */
-		sprintf(buf, "{\"ptr\":%"PRId64"}", (long)*(void **)v);
+		sprintf(buf, "{\"ptr\":%"PRId64"}", (uint64_t)*(void **)v);
```

Comment:
PRId64 required for int64_t
PRIu64 required for uint64_t

Reply:
fixed in 915ed2b